openedx/core/djangoapps/programs/tests/test_models.py

```diff
@@ -70,3 +71,51 @@ def test_is_student_dashboard_enabled(self, _mock_cache):
 
         self.create_config(enabled=True, enable_student_dashboard=True)
         self.assertTrue(ProgramsApiConfig.current().is_student_dashboard_enabled)
+
+    @ddt.data(
+        (True, True, 0),
+        (False, True, 0),
+        (False, True, 1),
+        (True, False, 0),
+    )
+    @ddt.unpack
+    def test_is_cache_enabled_returns_false(self, enabled, is_dashboard, ttl, _mock_cache):
+        """Verify that the method 'is_cache_enabled' returns false if
+        'cache_ttl' value is 0 or config is not enabled.
+        """
+        self.assertFalse(ProgramsApiConfig.current().is_cache_enabled)
+
+        self.create_config(
+            enabled=enabled,
+            enable_student_dashboard=is_dashboard,
+            cache_ttl=ttl
+        )
+        self.assertFalse(ProgramsApiConfig.current().is_cache_enabled)
+
+    @ddt.data(
+        (True, True, 1),
+        (True, False, 1),
+    )
+    @ddt.unpack
+    def test_is_cache_enabled_returns_true(self, enabled, is_dashboard, ttl, _mock_cache):
+        """Verify that the method 'is_cache_enabled' only returns True when the
+        current has 'enabled' and 'cache_ttl' has greater than 0 value.
+        """
+        self.create_config(
+            enabled=enabled,
+            enable_student_dashboard=is_dashboard,
+            cache_ttl=ttl
+        )
+        self.assertTrue(ProgramsApiConfig.current().is_cache_enabled)
+
+    @ddt.data(0, 1, 100, 200, 300)
```

Comment:
What's the value in doing this? All five values will return the same result. I think you only need to test this once.